cli/args/flags.rs

```diff
@@ -1548,9 +1552,10 @@ fn test_subcommand<'a>() -> Command<'a> {
       Arg::new("jobs")
         .short('j')
         .long("jobs")
-        .help("Number of parallel workers, defaults to # of CPUs when no value is provided. Defaults to 1 when the option is not present.")
+        .help("Number of parallel workers, defaults to number of available CPUs when no value is provided. Defaults to 1 when the option is not present.")
         .min_values(0)
         .max_values(1)
+        .require_equals(true)
```

Comment:
I think this is a breaking change, so we should leave it as-is and not require equals.

Reply:
Yep, I've added this to resolve the issue that now we can specify `--jobs` without a value now. So, if you write `deno test --jobs test.ts`, `test.ts` will be a value for `--jobs` flag. Do you have an idea on how this should be handled?

Reply:
Do we want to make this breaking change and fix the issue, or keep it as it was allowing to provide value with the equals sign?

Reply:
Hmmm... that is really annoying. I'm thinking we should just remove the deprecation notice on lines 2673-2676 then deprecate it once resolving #14533. So we would only be adding the DENO_JOBS env var in this PR. Sound ok?

Reply:
Sure, sounds good to me. I'll apply change to this PR accordingly.